app/components/agenda/agenda-header/agenda-actions.hbs

```diff
@@ -54,14 +54,32 @@
   >
     {{t "download-documents"}}
   </AuButton>
-    <AuButton
+  <AuButton
     data-test-agenda-actions-download-decisions
     @skin="link"
     {{on "click" this.openDownloadDecisions}}
     role="menuitem"
   >
     {{t "download-decisions"}}
   </AuButton>
+  {{#if this.allReports.length}}
```

Comment:
**question**: Should this action be restricted to certain profiles, since it adds to documents?

Reply:
If it writes to documents, only secretarie and admin